src/03-deals/deals.apib

```diff
@@ -36,8 +36,10 @@ Get a list of deals.
                 + field (enum, required)
                     + Members
                         + created_at
+                        + weighted_value
                 + order (enum, optional)
                     + Members
+                        + asc
```

Comment:
Q: why do we need ascending sorts for this story?

Reply:
We had a chat IRL and agreed that it's more work to not add ascending to the ordering.